client/me/account-close/main.jsx

```diff
@@ -81,12 +93,22 @@ class AccountSettingsClose extends Component {
 			hasCancelablePurchases,
 			isLoading,
 			purchasedPremiumThemes,
+			siteToBeDeleted,
 		} = this.props;
 		const isDeletePossible = ! isLoading && ! hasAtomicSites && ! hasCancelablePurchases;
 		const containerClasses = classnames( 'account-close', 'main', {
 			'is-loading': isLoading,
+			'is-hiding-other-sites': this.state.showSiteDropdown,
 		} );
 
+		/* eslint-disable-next-line no-shadow */
+		const listSites = siteToBeDeleted.map( siteToBeDeleted => (
+			<li key={ siteToBeDeleted.slug }>
+				{ [ siteToBeDeleted.name ] }
+				<span>{ [ siteToBeDeleted.slug ] }</span>
+			</li>
+		) );
+
 		return (
 			<div className={ containerClasses } role="main">
 				{ currentUserId && <QueryUserPurchases userId={ currentUserId } /> }
```

Comment:
I think we should possibly add a `<QuerySites allSites />` just in case we haven't loaded the user's sites when they get to this page (followed a direct link to /me/account/close perhaps)?

Reply:
`<QuerySites allSites />` is already done in [client/layout](https://github.com/Automattic/wp-calypso/blob/master/client/layout/index.jsx#L136) for all routes that don't explicitly opt out. It doesn't need to be added here.

Reply:
For what it's worth, it's also already done in this file.

https://github.com/Automattic/wp-calypso/blob/5670da208eb164bd448d42174c5967a05a25f339/client/me/account-close/main.jsx#L91

Reply:
Thanks for the information @jsnajdr! In which case, sounds like we can take it out of `me/account-close/index.jsx`.

Reply:
Done :)